app/models/player.rb

```diff
@@ -5,12 +5,14 @@ class Player < ApplicationRecord
   belongs_to :user, optional: true
   belongs_to :previous, class_name: 'Player', optional: true
   has_one :next, class_name: 'Player', foreign_key: :previous_id
+  belongs_to :corp_identity_ref, class_name: 'Identity', optional: true
```

Comment:
The fields {corp,runner}_identity already exist. It seems like we need to keep the string version in case someone wants to write in their own identity name. Adding _ref seems like the clearest way to differentiate the reference fields from the string fields.